src/main/java/appeng/tile/inventory/AppEngItemDefinitionFilter.java

```diff
@@ -0,0 +1,32 @@
+
+package appeng.tile.inventory;
+
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+import appeng.api.definitions.IItemDefinition;
+
+
+public class AppEngItemDefinitionFilter implements IAEItemFilter
+{
+	final private IItemDefinition definition;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)